### ISSUE TITLE
Update to Browserify 10.x

### ISSUE THREAD

**Issue by geirsagberg** [smrq/tsify#39]
Getting the following error:

```
npm ERR! peerinvalid The package browserify does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer tsify@0.8.1 wants browserify@6.x || 7.x || 8.x || 9.x
```

**Comment by bendrucker** [comment]
On a related note, I'll send a PR over adding https://github.com/bendrucker/test-peer-range when I finish it, probably later today. It basically automates the strategy we use over at browserify-shim to prevent regressions on old b versions.

**Comment by smrq** [comment]
It's that time again!

Given that the past 5 major versions of Browserify have worked just fine without any changes needed, I'm updating the peer dep to >= 6.x.  If there are problems down the line then I can always dial it back.  For now I'd rather err on the side of permissiveness.

@bendrucker I'm definitely interested to see that in action.  Sounds pretty sweet.

**Comment by smrq** [comment]
Released in v0.8.2.

**Comment by bendrucker** [comment]
Getting close to a 1.0. It actually handles unbounded peer deps by running tests all the way up to @latest. We've done it both ways on stuff that I've worked on. Really depends on how funky and public the APIs you're using are. If it hasn't been a problem you can definitely go unbounded.
